src/reviews/reviews.controller.ts

```diff
@@ -42,6 +43,7 @@ export class ReviewsController {
 
   @ApiBearerAuth()
   @Post()
+  @Throttle({ default: { limit: 3, ttl: 60000 } })
```

Comment:
@shakyaprasen Why are we overriding the defaults here? Do we want to limit the requests to just 3 in 60 seconds?

Reply:
Thank you for pointing this point out. Since, this is a POST request and this API has been abused previously. According to the our metrics, this API is used only once per customer so limiting it to 3 requests in case of API failures and to prevent ddos on this endpoint seems like a logical move.